ui/app/pages/send/send.utils.js

```diff
@@ -243,12 +244,21 @@ async function estimateGas ({
   }
 
   // if not, fall back to block gasLimit
+  if (!blockGasLimit) {
+    try {
+      blockGasLimit = await this.query.getBlockByNumber('latest', false)
```

Comment:
Won't `getBlockByNumber` return a block object, not a number string like we probably expect here?

Reply:
I believe it's impossible for `blockGasLimit` to ever be falsey, because it's given a default value of `MIN_GAS_LIMIT_HEX`

Reply:
I've updated this to something simpler. We don't really need a strictly accurate blockGasLimit at this point.

Meanwhile, I think `blockGasLimit` can be falsey if the value passed to this function is defined but falsey.

e.g.
```
danjm@pop-os:~/kyokan/metamask-extension$ node
> const t = ({ a = 10, b }) => console.log(`${a} | ${b}`)
undefined
> t({ a: null, b: 5})
null | 5
undefined
> t({ a: '', b: 5})
 | 5
undefined
> t({ b: 5})
10 | 5
undefined
```